src/content/docs/mobile-monitoring/new-relic-mobile/maintenance/configure-settings-for-mobile-monitoring.mdx

```diff
@@ -130,6 +130,10 @@ Server-side mobile agent configuration provides a mechanism to remotely manage a
 
 The following setting can be currently modified remotely on the **Application** page:
  * Application Not Responding (ANR) error tracking (Android only): Enables tracking and reporting of the reasons behind ANR errors. This functionality is enabled by default and requires Android agent 7.4.1 or higher.
+    <Callout variant="important">
+    Server-side configuration always take precedence over local agent configuration.
```

Comment:
I would add this at the end & link it - " For more information about how server-side configuration for mobile works , refer here (link the server side configuration in mobile monitoring settings here)

Reply:
@silpau-pm This will already be on the settings page (/docs/mobile-monitoring/new-relic-mobile/maintenance/configure-settings-for-mobile-monitoring)